bezier-rs/docs/interactive-docs/src/components/SubPathExamplePane.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+	<div>
+		<h3 class="example-pane-header">{{ name }}</h3>
+		<div class="example-row">
+			<div v-for="(example, index) in examples" :key="index">
+				<SubPathExample :title="example.title" :triples="example.triples" :closed="example.closed" :callback="callback" />
+			</div>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from "vue";
+
+import { SubPathCallback } from "@/utils/types";
+
+import SubPathExample from "@/components/SubPathExample.vue";
+
+export default defineComponent({
+	name: "SubPathPane",
+	components: {
+		SubPathExample,
+	},
+	data() {
+		return {
+			examples: [
+				{
+					title: "Open SubPath",
+					triples: [
+						[[20, 20], null, [10, 90]],
+						[[150, 40], [60, 40], null],
+						[[175, 175], null, null],
+						[[100, 100], [40, 120], null],
```

Comment:
Same story about `null` here but, in case wasm-bindgen provides `None` values as `null` (I haven't checked whether or not that's true), then it's okay strictly where necessary.